docs/gadgets/dns.md

```diff
@@ -1,10 +1,11 @@
-<!-- Code generated by 'make generate-documentation'. DO NOT EDIT. -->
-
-# Gadget dns
+---
+# Code generated by 'make generate-documentation'. DO NOT EDIT.
+title: Gadget dns
+---
 
 The dns gadget traces DNS requests.
 
-<h3 id="cr-example">Example CR</h3>
+### Example CR {#cr-example}
```

Comment:
It seems the id is visible in the markdown. Could it be removed completely?


![Screenshot from 2021-09-14 15-52-34](https://user-images.githubusercontent.com/27575/133270401-dc791a67-9914-485e-a9ac-f417f666aa2c.png)

Reply:
This is advanced markdown. Apparently Github doesn't process it.
Feel free to remove it if you want.

Reply:
I have removed it.